### DISCUSSION TITLE
[Feature Request] Display switch state in Web UI

### DISCUSSION THREAD

**Discussion by herangstl** [arendst/Tasmota#10787] [Ideas]
Please display the state of a Switch in the Tasmota Web UI (similar to the way sensor states are displayed).

A use case for this would be a garage door opener where I get the open/closed state via a reed switch. It would be quite handy to show this state in the Web UI.

**Comment by ascillato** [comment]
Duplicated Request from https://github.com/arendst/Tasmota/issues/9819

In the linked issue you have a driver for that if you want to try it and on work it.

**Comment by sfromis** [comment]
You could also create a virtual relay, which can be coupled with the switch input, and has the on/off status displayed in the web UI.
